### ISSUE TITLE
Blightfall opens with no issues but at the "Starting Game..." screen stops loading at 75%

### ISSUE THREAD

**Issue by kalebb95** [talonos2/BlightfallTracker#208]
The title explains the issue. I have searched everywhere but I have not been able to figure out what my problem is to fix this. I have tried uninstalling Technic, deleting from my app data, reinstalling Java, restarting my computer, watching youtube videos, and everything else that I can think of. I currently have 3GB of memory allowed for Blightfall (out of my 16gb total), I am using the latest Java 64bit, I am running Windows 10, and I have no issue with any other modpack from Technic or Feed the Beast.

http://pastebin.com/aruYT2ut

There is my entire Technic Launcher Console Window from the moment I opened it until my Minecraft 1.7.10 and Java(TM) Platform SE binary stopped responding.

Blightfall is on the Recommended Version (2.1.5) and the Java Version is Latest 64-Bit (1.8.0_77 64-Bit)

Please help me!!!!! :(

**Comment by talonos2** [comment]
This is not the log file that will help me. I need minecraft's log file, not the launcher's log file.

Send me Minecraft's log file, and I'll look through it.

**Comment by kalebb95** [comment]
Not 100% sure which Minecraft log file is needed, is it within the .technic file path? Under blightfall/logs? If so then I am guessing you need the 

fml-client-latest file: https://www.dropbox.com/s/6kmr35bog2mjhmg/fml-client-latest.log?dl=0

or the 2016-04-09-7.log file: http://pastebin.com/90aaGV5H

If you are needing another log file, I just need to know the name and/or location and I will get it uploaded ASAP. Thank you for the help!

**Comment by talonos2** [comment]
Huh. And you're saying it just hangs? It doesn't crash or anything? Hence
why there's no crash report?

**Comment by kalebb95** [comment]
Correct, it just sits there and tells me that it isn't responding. No crash, nothing shows up in the console window, nothing at all. I have tried everything I could find online and nothing has worked so far :(
